### PR TITLE
Revamp the footer

### PR DESCRIPTION
**Pull request by Ritik-Bhasarkar**
issue: #43
newfooter

### CODE REVIEW

**Comment by AdarshRawat1** [comment]
Make following changes 
- Bring "Open Code Era" and "improve this page on GitHub" to same level 
- Center align the copyright section 

OPTIONAL 
- Try to style "Open Code Era"

File: css/style_footer.css [hunk 1]
```diff
@@ -1,26 +1,84 @@
...
 }
-.footer ul{
-    text-decoration: none;
-    list-style: none;
+
+.footer ul {
+  list-style: none;
+  text-decoration: none;
 }
-footer .footer-rights{
```
Comment (AdarshRawat1) [inline css/style_footer.css:14]: Don't remove the styling of copyright section

**Comment by Ritik-Bhasarkar** [comment]
How about resizing the 'Open Code Era' and positioning it as it is in the left corner and the GitHub link to the right corner and copywrite to be at the center? Like the picture
footer#43

**Comment by AdarshRawat1** [comment]
> How about resizing the 'Open Code Era' and positioning it as it is in the left corner and the GitHub link to the right corner and copywrite to be at the center? Like the picture footer#43

looks good! Just reduce the height of the footer. 
We'll increase the height after adding the social icons.

**Comment by Ritik-Bhasarkar** [comment]
great. Working on it.

**Comment by Ritik-Bhasarkar** [comment]
Works?
image

**Comment by AdarshRawat1** [comment]
> Works? image

Nice work!

**Comment by AdarshRawat1** [comment]
Please make sure to commit your changes, @Ritik-Bhasarkar.

**Comment by AdarshRawat1** [comment]
Then we can merge it.

**Comment by Ritik-Bhasarkar** [comment]
@AdarshRawat1 The branch is ready to merge. Feel free to contact me if anything crashes. Happy to contribute and learn.

**Comment by AdarshRawat1** [comment]
@Ritik-Bhasarkar thanks for the PR

**Pull request merged**